web/app.js

```diff
@@ -176,6 +176,7 @@ var PDFViewerApplication = {
   preferencePdfBugEnabled: false,
   preferenceShowPreviousViewOnLoad: true,
   preferenceDefaultZoomValue: '',
+  preferenceEnhanceTextSelection: true,
```

Comment:
I agree here, this should be a preference. As a side note, I do think that enabling it by default is fine. Before merging this PR we must make sure we test it properly, but if we do not enable it by default most likely we not get a feedback for it.